lib/absinthe_error_payload/changeset_parser.ex

```diff
@@ -63,25 +64,25 @@ defmodule AbsintheErrorPayload.ChangesetParser do
     construct_message(field, {message, opts})
   end
 
-  defp construct_field(parent_field, field, options \\ []) do
+  def construct_field(parent_field, field, options \\ []) do
```

Comment:
Is this change related to the docs?

Reply:
I think this is a typo, I probably changed it by accident - reverted this change.